contentctl/objects/unit_test.py

```diff
@@ -1,16 +1,13 @@
 from __future__ import annotations
-from pydantic import Field
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Union
+
+# TODO (cmcginley): what's the purpose of this conditional?
```

Comment:
What're we doing here w/ these conditional imports? Esp. bc we import them again a few lines later

Reply:
Good catch. These should not be double-imported.
The conditional imports are because many of our objects are structured...interesting and _intentionally_ do circular imports.  For example, a Story includes a list of Detections and a Detection includes a list of Stories.

For typing reasons, Story needs to import Detection and Detection needs to import Story - but Python/Pydantic throw parse exceptions if you don't import them (as expected). However, a regular import throws a _different_ runtime Pydantic exception about models not being fully defined.

Putting them imports in an if `TYPE_CHECKING` block satisfies the type checker and avoids the runtime errors mentioned above. It's a bit of a mess - we may be able to do this better by _explicitly_ calling `PydanticObject.model_rebuild` after all objects have been imported, but I haven't been able to resolve that yet.